pkgs/tools/security/age/default.nix

```diff
@@ -18,6 +18,19 @@ buildGoModule rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
+  preCheck = ''
+    # Commit bf8d2a[*] moved test data used by TestVectors test from age's
+    # repository to C2SP's CCTV repo.  Implementation attemps to download these
+    # test time with `go mod download`, requiring network access.
+    #
+    # Temporarily disable this test while we try to work with the upstream to
```

Comment:
Is there an upstream issue you filed for this? If yes, might be useful to link it here to track that

Reply:
Also, can we just reconstruct that tree by fetching those sources in a separate drv and "injecting" them in the right place? (I haven't really looked at the test code, so the answer might be "obviously no")

Reply:
Looks like we would need to patch the test file to do the injection: https://github.com/FiloSottile/age/blob/main/testkit_test.go#L35

IMO an upstream change would be the best approach.

Reply:
I did have time to talk to Filippo yet.

Updated the PR to pull in the testdata from C2SP/CCTV@cfaa74dc42af as a derivation, patch the test to get the path to testdata from an environment variable.